### ISSUE TITLE
The app is not working properly gor our website

### ISSUE THREAD

**Issue by Optimus2297** [responsively-org/responsively-app#597]
### ✍️ Description

The app is not working properly for our website , its built using polymer, its a product of our company so no more info can be provided

### 🔬 Minimal Reproduction

The issue is if i login in one device it doesnt get synced in all other devices

### 🌍 Your Environment

polymer, node.js

**Comment by ganapativs** [comment]
Similar issue with https://sgb.vercel.app/. It renders perfectly in Chrome, breaks in desktop and mobile mode when rendering through responsively. Looks like the CSS flex gap property is not working.

**Comment by manojVivek** [comment]
@ganapativs `flex-gap` issue is tracked with #582.

**Comment by Optimus2297** [comment]
> @ganapativs `flex-gap` issue is tracked with #582.

so how can it be resolved

**Comment by Optimus2297** [comment]
> @ganapativs `flex-gap` issue is tracked with #582.

But we are not using flex-gap in our application. so what might be causing this, we are passing url parameters in the url.

**Comment by jjavierdguezas** [comment]
Hi @Optimus2297, could you please provide more information on what is not working properly? Any information could help us understand what your problem is. Ideally, if you can provide a sample app where we can reproduce it, the better

**Comment by jjavierdguezas** [comment]
>Also if possible can you explain how the event mirroring is implemented

We use [`browser-sync`](https://browsersync.io/)

**Comment by Optimus2297** [comment]
> Hi @Optimus2297, could you please provide more information on what is not working properly? Any information could help us understand what your problem is. Ideally, if you can provide a sample app where we can reproduce it, the better

sorry @jjavierdguezas its a product, so cant give any more info

**Comment by Karthick007** [comment]
In Polymer it uses shadow root so events are not communicated across the emulated devices. Live CSS editor not selecting the DOM inside the shadow root elements.

**Comment by Optimus2297** [comment]
> In Polymer it uses shadow root so events are not communicated across the emulated devices. Live CSS editor not selecting the DOM inside the shadow root elements.

yes, the application is in polymer, so is there any way to fix this issue?
